share/spice/clojars/clojars.js

```diff
@@ -0,0 +1,52 @@
+(function(env) {
+    "use strict";
+    var search_terms = DDG.get_query().split(" ");
+
+    function keep_it(candidate) {
+        return DDG.isRelevant(candidate, ['clojure', 'clojars', 'lib', 'library','pacakge'], 1, 0);
+    }
+
+        var clojar_artifact = search_terms.filter(keep_it);
+    env.ddg_spice_clojars = function(api_result) {
+
+        // Validate the response (customize for your Spice)
+      
+
+        if (!api_result || api_result.error) {
```

Comment:
Right, there's no such thing as api_result.error. Instead, I've put api_result.count, which when shows 0, means that no result was shown.